Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +494,107 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         return [delete]
     }
 }
+
+class SyncedTabsButton: UIView {
```

Comment:
If we're using this as a button why not just subclass UIButton? It would save you from having to add a custom gesture recognizer. If you want to keep it a view I'd rename this `SyncedTabsHeaderView` or something since button makes me think it's a UIButton.